### PR TITLE
2 - API can serve a single article

### PR DESCRIPTION
**Pull request by ViaMarcus**
[PT LINK](https://www.pivotaltracker.com/story/show/172913649)
In this PR:
- Adds :body:text to articles model/db table
- Adds :show action to articles controller
- Adds serializer for single article, and refactors serializer namespace
- Sad path rescue for when id is invalid

Please see README for API endpoint details

### CODE REVIEW

File: spec/requests/api/articles_request_spec.rb [hunk 1]
```diff
@@ -17,9 +17,44 @@
...
 
     describe 'response has keys' do
-      it 'has key :title' do
+      it ':title' do
         expect(response_json['articles'][0]).to have_key 'title'
       end
     end
   end
+
+  describe 'GET /api/articles/:id' do
```
Comment (oliverochman) [inline spec/requests/api/articles_request_spec.rb]: Create a article folder within the request folder. Have a index_spec and a show_spec. 

We dont want to test these two different action in the same spec file.

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: spec/requests/api/articles_request_spec.rb (hunk 1 above)
Comment (ViaMarcus) [reply spec/requests/api/articles_request_spec.rb]: Will refactor

Comment (ViaMarcus) [reply spec/requests/api/articles_request_spec.rb]: Refactor done

**Review by erikbjoern — APPROVED** [review]
All namings and typos fixed, all good.

**Review by kermit-klein — APPROVED** [review]

**Pull request merged**